scripts/koji/.python-version

```diff
@@ -0,0 +1 @@
+3.11.11
```

Comment:
What's the point of this file? This version can't be enforced (my system doesn't provide it.

Reply:
`uv` uses it to pin the python version

Reply:
Feels strange, as https://docs.astral.sh/uv/guides/tools/ says `uv tool run` is the same as `uvx`, and
> Tools are installed into temporary, isolated environments when using uvx.